physics/module_bl_mynn.F90

```diff
@@ -2215,20 +2085,20 @@ SUBROUTINE  mym_length (                     &
         Uonset = 3.5 + dz(kts)*0.1
         Ugrid  = sqrt(u1(kts)**2 + v1(kts)**2)
         cns  = 3.5 !JOE-test  * (1.0 - MIN(MAX(Ugrid - Uonset, 0.0)/10.0, 1.0))
-        alp1 = 0.21
+        alp1 = 0.22
         alp2 = 0.30
-        alp3 = 1.5
+        alp3 = 2.0
         alp4 = 5.0
         alp5 = alp2 !like alp2, but for free atmosphere
         alp6 = 50.0 !used for MF mixing length
 
         ! Impose limits on the height integration for elt and the transition layer depth
         !zi2=MAX(zi,minzi)
-        zi2=MAX(zi,    200.)
+        zi2=MAX(zi,    300.)
```

Comment:
ditto for these max/mins

Reply:
I don't follow the ditto. Maybe the sequence of the comments I'm reading is different from the order at which you made them?

Reply:
I think that I am referring to the opportunity to define min/max variables/parameters rather than hard-coded values. It's completely up to you, but I typically point this out in reviews when hard-coded values change in PRs because it can signal that the hard-coded values are really tuning knobs deserving of making them variables/parameters. If this is a one time change, then, obviously, there is no need.